modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -730,20 +821,19 @@ struct InferROI: public cv::detail::KernelTag {
         const auto &uu = gm.metadata(nh).get<ONNXUnit>();
         GAPI_Assert(1u == uu.oc->numInputs());
         GAPI_Assert(2u == in_metas.size());
-
+        checkInputMeta(in_metas.at(1));
         for (auto &&idx : ade::util::iota(uu.oc->numOutputs())) {
             result.emplace_back(uu.oc->outMeta(idx));
         }
         return result;
     }
 
     static void run(const ONNXUnit &uu, ONNXCallContext &ctx) {
```

Comment:
Is it the same function as on 797 line?

Reply:
What's the point of that and how do you distinguish them? Maybe I just missing the namespace/class/context

Reply:
There are more below